src/App.tsx

```diff
@@ -18,6 +18,7 @@ import NotFound from './pages/NotFound'
 import AdministratorManagement from './pages/AdministratorManagement'
 import DashboardSelection from './pages/DashboardSelection'
 import { ProtectedRoute } from './components/ProtectedRoute'
+import GradesClosing from './pages/GradesClosing'
```

Comment:
"Grades closing" nao é tradução para "Fechamento de notas", melhor usar Grades Management
```suggestion
import GradesManagement from './pages/GradesManagement'
```